src/service/resources-impl.js

```diff
@@ -141,6 +141,9 @@ export class Resources {
     /** @private {!Array<!Function>} */
     this.deferredMutates_ = [];
 
+    /** @private {!Array<!Element>} */
```

Comment:
Let's do `?` so that we can reset it after we no longer collecting open elements. This way it'd be easy to catch any regressions if anyone is trying to use this array after.